resources/keywords/import_provider_account.resource

```diff
@@ -294,13 +301,16 @@ Provider Account Import Failure Using CLI
 Duplicate Provider Account Name Error Message Appears
     Element Should Be Visible    ${lblDuplProvAccName}
 
-User Navigates To Add ${databaseProvider} Topology Screen From Developer View
+User Navigates To Add Topology Screen From Developer View
+    [Arguments]     ${databaseProvider}     ${ns}=${operatorNamespace}      ${developer}=False
     User Selects Developer View
-    User Filters ${operatorNamespace} Namespace On Project Dropdown
+    User Filters ${ns} Namespace On Project Dropdown
     User Navigates To Developers Catalog Add Screen
-    User Clicks On Cloud Hosted Database Menu
-    User Selects ${databaseProvider} Tile Under Cloud Hosted Database
-    Click Element    ${btnDBProvAddTopology}
+    IF  "${developer}"=="False"
```

Comment:
The ${developer} parameter here relates to DBaaS Policy? If not, I think it is not necessary because we are already under developer view when we run line 306

Reply:
The ${developer} parameter indicates DBaaSPersona here, so if the user logged in with the Developer level access, the steps defined in the conditional statements would be skipped.

Reply:
OK